src/main/webapp/i18n/de/exam.json

```
@@ -268,6 +268,15 @@
             "updated": "Aufgabengruppe bearbeitet",
             "deleted": "Aufgabengruppe mit Titel {{ param }} gelöscht"
         },
+        "examSession": {
+            "id": "ID",
+            "sessionToken": "Zeichen",
+            "userAgent": "Nutzer Agent",
```

```suggestion
            "userAgent": "User Agent",
```
I don’t think there is a proper translation for that.